firefox-ios/Providers/Profile.swift

```diff
@@ -771,7 +771,7 @@ open class BrowserProfile: Profile {
         }
         var customTargetingAttributes: String?
         let customTargetingAttributesData = try? encoder.encode([
-                "form_factor": formFactor,
+                "formFactor": formFactor,
```

Comment:
Nice!

Should we add or look into adding tests to catch this?

I don't know where these would live but could avoid failure in future

Reply:
We definitely need a test here. I will add a ticket for this. We can discuss async where it should live.